P5/Source/Specs/provenance.xml

```diff
@@ -3,6 +3,7 @@
 <?xml-model href="https://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?><elementSpec xmlns="http://www.tei-c.org/ns/1.0" xmlns:sch="http://purl.oclc.org/dsdl/schematron" module="msdescription" xml:id="PROVENANCE" ident="provenance">
   <gloss xml:lang="en" versionDate="2007-06-12">provenance</gloss>
   <gloss versionDate="2007-06-12" xml:lang="fr">provenance</gloss>
+  <gloss versionDate="2023-08-08" xml:lang="de">Provenienz</gloss>
   <desc versionDate="2019-01-17" xml:lang="en" xml:id="prov.desc">contains any descriptive or other information
 concerning a single identifiable episode during the history of a manuscript, manuscript part, or other object after its creation but before its acquisition.</desc>
```

Comment:
The translation of the gloss is correct but I wonder a bit about the definition: Why is the phrase "concerning a single identifiable episode" in there? Provenance means the whole history of a manuscript/object (that, yes, consists of several single identifiable episodes). I am also not sure about the emphasis on manuscripts as they are one example of so many objects that can have a provenance.
Should we do something about this definition?

Reply:
The tei element provenance by definition describes a single significant episode in the life of an object. The element for describing several such episodes is history.

Reply:
In German "Provenienz" means the whole history of one object. Therefore, to be correct, one should translate the TEI element `history` with the gloss "Provenienz" (which might be confusing) and the TEI element `provenance` with --? I am not sure.

Reply:
tei:history has three components : information about the origin of the item, information about its acquisition by  the place where it now resides, and information about anything that happened between the two. Is "Provenienz" an appropriate German term for all three of these?

Reply:
"Provenienz" is in German often used as term for the collection of all provenances, but still it only means a "single identifiable episode" in the object's history.

Reply:
I would agree: "Provenienz" could mean both the whole history of an object as well as a single identifiable episode. In German we would distinguish between "the provenance" and "a provenance". To complicate it even further, in manuscript cataloguing, "Provenienz I" represents the origin, whereas "Provenienz II" represents the single identifiable episodes.

Reply:
Suggestion: Perhaps we can translate ´history` with 'Provenienzgeschichte' and ´provenance´ with 'Provenienz'?

Reply:
Maybe, but that wouldn't affect this PR, would it?

Reply:
This is true as I am now ok with translating `provenance` with "Provenienz" :-)